Subject: Linux-RT 2.6.25-rt
From: http://www.kernel.org/pub/linux/kernel/projects/rt/
Acked-by: Sven-Thorsten Dietrich <sdietrich@suse.de>
From: Steven Rostedt <rostedt@redhat.com>
Subject: Add context switch marker to sched.c

Add marker into context_switch to record the prev and next tasks.

Signed-off-by: Steven Rostedt <srostedt@redhat.com>
---
 kernel/sched.c |    2 ++
 1 file changed, 2 insertions(+)

Index: linux-2.6.24-rt2/kernel/sched.c
===================================================================
--- linux-2.6.24-rt2.orig/kernel/sched.c	2008-01-30 16:43:34.000000000 -0500
+++ linux-2.6.24-rt2/kernel/sched.c	2008-01-30 16:43:37.000000000 -0500
@@ -1882,6 +1882,8 @@ context_switch(struct rq *rq, struct tas
 	struct mm_struct *mm, *oldmm;
 
 	prepare_task_switch(rq, prev, next);
+	trace_mark(kernel_sched_schedule,
+		   "prev %p next %p", prev, next);
 	mm = next->mm;
 	oldmm = prev->active_mm;
 	/*
